src/Examples/Examples/Root/MappingPage.cs

```diff
@@ -1,18 +1,27 @@
 using Elastic.Xunit.XunitPlumbing;
-using Nest;
+using Examples.Models;
+using static Nest.Infer;
 
 namespace Examples.Root
 {
 	public class MappingPage : ExampleBase
 	{
-		[U(Skip = "Example not implemented")]
+		[U]
 		public void Line144()
 		{
 			// tag::d8b2a88b5eca99d3691ad3cd40266736[]
-			var response0 = new SearchResponse<object>();
+			var mapResponse = client.Indices.Create("my-index", c => c
+				.Map<Employee>(m => m
+					.Properties(props => props
+						.Scalar(p => p.Age)
```

Comment:
Do we want to get rid of `Scalar` in 8.0.0?

Reply:
Aye I like the field name first approach of `Scalar` but I think we can do better in terms of extensiblity